input/pagecontent/overview.md

```diff
@@ -283,51 +283,32 @@ The [formal syntax of rules](reference.html#rules-for-profiles-extensions-logica
+<img src="Workflow.png" alt="Overall FSH Workflow" width="800px" style="float:none; margin: 0px 0px 0px 0px;" />
 
-SUSHI can be used in two modes:
+#### Preparing FSH Files
 
-* A [standalone mode](#sushi-standalone-mode) where SUSHI produces FHIR artifacts only.
-* A [IG mode](#sushi-ig-mode) where SUSHI works together with the [HL7 FHIR IG Publisher](https://confluence.hl7.org/display/FHIR/IG+Publisher+Documentation). The IG Publisher builds the human-readable web pages and bundles FHIR artifacts produced by SUSHI to produce an Implementation Guide.
+Content written in FSH is stored in plain text files (ASCII or UTF-8) with the `.fsh` extension (1). Any text editor can be used to create a FSH file. [Visual Studio Code](https://code.visualstudio.com/) has a useful [FSH plug-in](https://marketplace.visualstudio.com/search?term=fhir%20shorthand&target=VSCode&category=Programming%20Languages&sortBy=Relevance) that knows FSH syntax and colorizes text accordingly.
 
-#### SUSHI Standalone Mode
+SUSHI lets the author decide how to allocate FSH definitions to FSH files. Here are some possibilities:
 
-In the standalone mode, SUSHI runs independently of the IG Publisher. This is a good option to quickly check for errors when you are creating your FSH code, or if you are only interesting in creating FHIR artifacts without an IG. Creating FHIR artifacts with FSH and SUSHI in standalone mode involves the following steps:
+* One file per item
+* All profile definitions in one file, all value set definitions in another, all extensions in another, etc.
+* Group related items in one file, e.g., a profile together with its value sets, extensions, and examples
+* Create subdirectories for each type of item (profiles, extensions, value sets), with separate files for each item of the corresponding type inside those subdirectories.
 
-1. Populate a FSH Tank (2) with FSH files (1) containing your FSH definitions.
-2. Create a **[sushi-config.yaml](https://fshschool.org/docs/sushi/configuration/)** file (3).
-3. Run SUSHI (4). After SUSHI runs, a new directory (named **/fsh-generated** by default) appears in the FSH Tank. This directory contains FHIR artifacts (5) such as profiles, extensions, value sets, and instances.
+#### Running SUSHI
 
-#### SUSHI IG Mode
+Before running SUSHI, you must create a [configuration file named **sushi-config.yaml**](https://fshschool.org/docs/sushi/configuration/) (2) containing some basic information about the project, such as its canonical URL.
 
-Creating an IG with FSH and SUSHI involves the following steps:
+When SUSHI runs (3), it gathers all FSH files from the hierarchy of a given input directory (1), and writes generated FHIR JSON artifacts to a given output directory (4). The input and output directories appear as SUSHI command line arguments. If unspecified, the input directory will default to the current directory, and outputs will be written to **./fsh-generated**.
```

Comment:
> When SUSHI runs (3), it gathers all FSH files from the hierarchy of a given input directory (1)

Whatever the input directory you pass into SUSHI, it will only gather FSH files from the **input/fsh** subpath within that directory. The graphic shows (1) as **input/fsh**, but I feel like the above text implies it will recursively visit the whole directory structure, which is not the case since it ignores anything outside of **input/fsh**.  When you mention **input/fsh** below, it sounds like it might only apply when SUSHI is invoked by the IG Publisher -- but it now applies in all cases (IG Publisher or not).

I think we can clarify the text a bit more.  Might something like this be better?
> When SUSHI runs (3), it gathers all FSH files from the input directory's **input/fsh** subdirectory (1)